src/PkgServer.jl

```diff
@@ -221,7 +221,7 @@ function serve_file(http::HTTP.Stream, path::String)
     end
 end
 
-function start()
+function start(;host=Sockets.localhost, port=8000)
```

Comment:
Couldn't this just be `127.0.0.1` and avoid the dependency on `Sockets`?

Reply:
I assumed that the whole point of `Sockets.localhost` was that it would be fancy and use `::1` if you were on an ipv6-only host, but apparently that is not the case.